build.gradle

```diff
@@ -240,13 +240,14 @@ subprojects {
             targetCompatibility = 1.8
         }
 
-        tasks.withType(AbstractCompile).all {
+        tasks.matching { it instanceof JavaCompile || it instanceof GroovyCompile }.all {
             // enable all warnings except for different sourceCompatibility and targetCompatibility value
             options.compilerArgs << '-Xlint:all'
             options.compilerArgs << '-Xlint:-options'
-            if (targetPlatformVersion >= '46') {
-                options.compilerArgs << '-Werror'                
-            }
+// TODO (donat) kotlin compilation currently fails with bad path error
```

Comment:
I got the following Java compilation error which I wanted to deal with later:
`warning: [path] bad path element "/Users/donat/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-reflect/1.1-M04/50e442efdd3eab1dcf2ddbf4b3ec5dabecf1b2b2/kotlin-runtime.jar": no such file or directory`